docs/Modpacks/Materials-and-Elements/01-Material-Creation.md

```diff
@@ -3,6 +3,74 @@ title: Material Creation
+
+- .iconSet(set) -> gives the material an icon set.
+- .components(component1, component2, ...) -> describes the composition. The components are a list of elements of the following form: 'Kx material_name', where K is a positive integer.
+
+Depending on the composition, gt will autogenerate an electrolyzer or centrifuge recipe to decompose the material. You can block that by adding the disable decomposition flag.
+
+- .flags(flag1, flag2, ...) -> flags can be used to select certain properties of the material, like generating gears, or disabling decomposition.
+- .element(element) -> similar to .components(), but is used when the material represents an element.
+- .rotorStats(speed, damage, durability) -> this will create a turbine rotor from this material.
+- .blastTemp() -> this is meant to be paired together with .ingot(). Will generate a EBF recipe(and an ABS recipe) based on the parameters you give it:
+    1. temperature -> dictates what coil tier it will require(check the coil tooltips for their max temperature).
+        If the temperature is below 1000, it will also generate a PBF recipe.
+        If temperature is above 1750, a hot ingot will be generated, this requiring a Vacuum Freezer.
+    2. (optional) gas tier -> can be null for none, 'low' for nitrogen, 'mid' for helium, 'high' for argon, 'higher' for neon or 'highest' for krypton.
+    3. (optional) EU per tick -> the recipe voltage
+
+**USEFUL NOTE**: GT has some inbuilt functions to ease choosing the voltages, you can use GTValues.V(GTValues.[voltage tier]) for full amp, GTValues.VA(GTValues.[voltage tier]) for full amp, but adjusted for cable loss, GTValues.VH(GTValues.[voltage tier]) for half an amp or GTValues.VHA(GTValues.[voltage tier]) for half an amp adjusted for cable loss.
+
+4. (optional) duration in ticks -> how long the recipe should take
+
+- .ore() -> this will create an ore from the material.
+
+Optionally you can add any of these sets of parameters: 1. is emissive -> true for emissive textures 2. ore multiplier and byproduct multiplier -> how many crushed ores will be given from one raw ore and how many byproducts dusts will be given throughout the ore processing 3. ore multiplier, byproduct multiplier, is emissive
+
+- .washedIn()
+- .separatedIn()
+- .separatedInto()
+- .oreSmeltInto()
+- .polarizesInto()
+- .arcSmeltInto()
+- .maceratesInto()
+- .ingotSmeltInto()
+- .addOreByproducts()
+- .cableProperties() -> generates wires and cables(if material is not a superconductor). The following parameter sets can be given:
+    1. Voltage, amperage, loss per block
+    2. Voltage, amperage, loss per block, is superconductor -> for a super conductor, set loss as 0 and is super conductor as true
+    3. Voltage, amperage, loss per block, is super conductor and critical temperature
+
+- .toolProperties()
+- .fluidPipeProperties()
+- .itemPipeProperties()
+- .addDefaultEnchant()
+
```

Comment:
Forgot to mention: An alternative approach would be making each option its own section in case you want to keep the ordering like it was initially.

Whichever you want to use is fine ^^